boards/atxmega-a3bu-xplained/include/periph_conf.h

```diff
@@ -129,6 +129,24 @@ static const i2c_conf_t i2c_config[] = {
 #define I2C_NUMOF           ARRAY_SIZE(i2c_config)
 /** @} */
 
+/*
```

Comment:
```suggestion
/**
```

Reply:
unbelievable...fixed!
Sorry about that.